NineChronicles.Headless.Executable/Store/StoreTypeExtensions.cs

```diff
@@ -8,7 +8,7 @@ public static class StoreTypeExtensions
     {
         public static IStore CreateStore(this StoreType storeType, string storePath) => storeType switch
         {
-            StoreType.RocksDb => new RocksDBStore(storePath),
+            StoreType.RocksDb => new RocksDBStore(storePath, dbConnectionCacheSize: 5),
```

Comment:
Although it's obviously a good short-term fix, could you change the default value of `RocksDBStore()` constructor's `dbConnectionCacheSize` too?